src/sink/wavefront.rs

```diff
@@ -392,7 +570,7 @@ mod test {
         wavefront.format_stats(dt_2);
         let lines: Vec<&str> = wavefront.stats.lines().collect();
 
-        println!("{:?}", lines);
+        //println!("{:?}", lines);
```

Comment:
Accidental comment. This is useful in test as the lines are only printed out on failure. Will correct.